Data/Sys/GameSettings/RM3.ini

```diff
@@ -19,7 +19,7 @@ EmulationIssues =
 
 [Video_Settings]
 SafeTextureCacheColorSamples = 512
-UseRealXFB = True # Prevent black bar
```

Comment:
You should also set userealxfb to false instead of removing it. As it is xfb will be enabled but whether real or virtual is used will depend on what the user have set it last time in the general settings.